less/addEditBookmark.less

```diff
@@ -2,6 +2,16 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+@import "variables.less";
+
 .removeBookmarkLink {
   margin-right: auto;
 }
+
+.invalid-input {
+    border: @errorBorderSize solid @errorBorderColor;
+}
+
+.disabled {
+    background-color: #ccc!important;
```

Comment:
nit: add a space before  `!important` or just be more specific and then you can bypass the need for it altogether.
background-color: #ccc !important;